src/Symfony/Component/Notifier/Bridge/Firebase/composer.json

```diff
@@ -2,7 +2,10 @@
     "name": "symfony/firebase-notifier",
     "type": "symfony-bridge",
     "description": "Symfony Firebase Notifier Bridge",
-    "keywords": ["firebase", "notifier"],
+    "keywords": [
+        "firebase",
+        "notifier"
+    ],
```

Comment:
I think we have all keywords on the same line everywhere else, I would keep it that way.